.gitlab-ci.yml

```diff
@@ -139,9 +139,6 @@ test-performance:
     - $DNF_WITH_OPTIONS install $RPM_REQUIREMENTS
     - python3.9 -m pip install tox
     - tox -e corgi -- -m performance --no-cov
-  only:
```

Comment:
I don't agree with this. I fixed the performance test in stage by regenerating the rhel-9.2.0 manifest. I guess the issue might have been caused by changes James was testing for the the latest_components stored procedure.

The reason we removed it was because it's testing changes which aren't from the current branch under test. It's annoying to have it fail when the changes are not related to what you are trying to merge.

Reply:
For me, I visit the pipelines page of Gitlab often enough that I think I would notice if the scheduled task starting failing, and I would go and investigate that. But if you think that isn't good enough, I guess we can re-enable the performance test in CI.